docker_readme.txt

```diff
@@ -0,0 +1,9 @@
+Steps for Docker: 
```

Comment:
Add this text to the README.md file instead. GitHub will present it on the repo main page.